### DISCUSSION TITLE
How to disable keyring detection and use --noprofile?

### DISCUSSION THREAD

**Discussion by neurodiverseEsoteric** [netblue30/firejail#4496] [Q&A] [answered]
Electron apps like Biscuit open with a popup dialog for a global keyring password, that won't allow interaction with anything else on the desktop until hitting esc/return for that dialog...I tried 
` seccomp=@keyring,!chroot,`  which didn't work; nor did `--deny=~./pki`  which threw an invalid command line option error. Is there any way to stop this from happening?

(*I was only able to get the printscreen key to work after pressing both it and esc at the same time, since the prompt happens twice:)

![keyringNonsense](https://user-images.githubusercontent.com/577091/131436406-2eb378d1-3411-49fd-a2b1-729e65d5d1cf.jpg)

**Comment by rusty-snake** [comment, marked as answer]
To block access to the secret-service, you can use `--dbus-user=none` (or `--nodbus`).

**Reply by neurodiverseEsoteric** [reply]
Thanks, "`--nodubs`" seems to have done the trick!

**Reply by neurodiverseEsoteric** [reply]
(...For some reason I'm not allowed to activate "mark as answer...")

**Reply by rusty-snake** [reply]
Wait GH does not all the OP user to select the answer that worked (best) for him/her. LOL.
I mean a moderate function if a user does not select an answer (that is correct) is good, but the OP users knowns it the best.

**Reply by kmk3** [reply]
> (...For some reason I'm not allowed to activate "mark as answer...")

Seems to be a GitHub bug:

https://github.com/github/feedback/discussions/5242

Note: I have asked a question and successfully used "mark as answer" in there
before.

**Comment by kmk3** [comment]
@neurodiverseEsoteric [on Aug 31](https://github.com/netblue30/firejail/discussions/4496#discussion-3547542):

> nor did `--deny=~./pki` which threw an invalid command line option error.

I don't know about the error, but I'd recommend using just
`whitelist`/`blacklist` instead of any aliases until we come to a conclusion
about what they will be.

Sorry about the confusion; there are some leftovers since #4410.  I'll see if I
can fix that in another PR.

**Reply by rusty-snake** [reply]
@neurodiverseEsoteric `=~./pki` -> `=~/.pki`? Anyway that's not the secret-service keyring.

**Reply by neurodiverseEsoteric** [reply]
Oh...then is it not a home directory/file thing?
